src/chainparams.cpp

```diff
@@ -88,10 +88,10 @@ class CMainParams : public CChainParams {
 
         // Deployment of Taproot (BIPs 340-342)
         consensus.vDeployments[Consensus::DEPLOYMENT_TAPROOT].bit = 2;
-        consensus.vDeployments[Consensus::DEPLOYMENT_TAPROOT].startheight = Consensus::VBitsDeployment::NEVER_ACTIVE;
-        consensus.vDeployments[Consensus::DEPLOYMENT_TAPROOT].timeoutheight = Consensus::VBitsDeployment::NEVER_ACTIVE;
-        consensus.vDeployments[Consensus::DEPLOYMENT_TAPROOT].threshold = 1916; // 95% of 2016
-        consensus.vDeployments[Consensus::DEPLOYMENT_TAPROOT].m_min_activation_height = 0; // No minimum activation height
+        consensus.vDeployments[Consensus::DEPLOYMENT_TAPROOT].startheight = 681408;
+        consensus.vDeployments[Consensus::DEPLOYMENT_TAPROOT].timeoutheight = 695520;
```

Comment:
The last period before timeoutheight here overlaps with the current BIP8(True) deployment plan. So if this period specifically were to reach 90% signalling, nodes would activate Taproot at height 697536, but ST-only nodes would still wait until 709632 instead.

Reply:
The [data](https://gist.github.com/michaelfolkson/92899f27f1ab30aa2ebee82314f8fe7f) suggests there is considerable more consensus around ST than BIP8(True). Therefore I think the onus is on the UASF release to fit around the ST release rather than vice versa.